NOTICE.txt

```diff
@@ -4,6 +4,9 @@ Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 This product includes software developed by
 Amazon Technologies, Inc (http://www.amazon.com/).
 
+This product includes software developed at

```

Comment:
It needs the full text. So looking at their latest, it reads:

```
Apache Commons Lang
Copyright 2001-2020 The Apache Software Foundation

This product includes software developed at
The Apache Software Foundation (https://www.apache.org/).
```

I would also place it lower down in the document either under the bullet where you mention Commons Lang, or under the bullets in general. For example, for the under in general:


* For Apache Commons Lang see also this required NOTICE:
    Apache Commons Lang
    Copyright 2001-2020 The Apache Software Foundation

    This product includes software developed at
    The Apache Software Foundation (https://www.apache.org/).

Reply:
Thanks for the pointer. Updated the text